src/components/NewTask.tsx

```diff
@@ -33,11 +48,9 @@ const LocaleCheckbox = ({ locale, toggle, checked }: LocaleCheckboxProps) => {
           //not needed because of above toggle logic, but silence React warnings.
           onChange={() => {}}
         />
-        <Box>
-          <Text size={1} weight="semibold">
-            {locale.description}
-          </Text>
-        </Box>
+        <WrapText>
+          <Text size={1}>{locale.description}</Text>
```

Comment:
```suggestion
 <Text size={1} weight="semibold">{locale.description}</Text>
 ```
 
 A bit light in default styling, would like to keep it readable.

Reply:
Sorry I don't think the "semibold" suggestion made it into the code, would you mind applying it?